lib/web_ui/test/canvaskit/canvaskit_api_tt_on_test.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test/bootstrap/browser.dart';
+import 'package:test/test.dart';
+import 'package:ui/src/engine.dart';
+
+import '../matchers.dart';
+import 'canvaskit_api_test.dart';
+
+// These tests need to happen in a separate file, because a Content Security
+// Policy cannot be relaxed once set, only made more strict.
+void main() {
+  enableTrustedTypes();
+  internalBootstrapBrowserTest(() => () {
+    // Run all standard canvaskit tests, with TT on...
+    testMain();
+
+    test('rejects wrong canvaskit.js URL', () async {
+      expect(() {
+        createTrustedScriptUrl('https://www.unpkg.com/soemthing/not-canvaskit.js');
+      }, throwsAssertionError);
+    }, skip: isSafari || isFirefox);
```

Comment:
Should we have a Safari/Firefox test that checks what happens in those browsers?

Reply:
Added, also added a test for the case where the actual TrustedScriptURL *is* created.